docs/containers-registries.conf.5.md

```diff
@@ -108,16 +108,24 @@ Each TOML table in the `mirror` array can contain the following fields, with the
 as if specified in the `[[registry]]` TOML table directly:
 - `location`
 - `insecure`
+- `digest-only`
```

Comment:
The new text is fine, but there’s another entry for `digest-only` that is now unnecessary or even misleading (it suggests that it can be set on `[[registry]]`.